components/automate-gateway/api/iam/v2/tokens.proto

```diff
@@ -26,7 +47,16 @@ service Tokens {
     option (chef.automate.api.policy).action = "create";
     option (chef.automate.api.iam.policy).resource = "iam:tokens";
     option (chef.automate.api.iam.policy).action = "iam:tokens:create";
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      tags: "tokens"
+    };
   };
+
+  /*
+  Get a token
+
+  Returns the details for a token.
```

Comment:
I commented in the policy PR that the docs looked a bit broken without a one-line like this (though it does not really add value otherwise). You add it for `get` here, but not for `delete` or `list` below. If you want to argue for not having it, that is fine, too; my preference is not a strong preference. But again, let's be consistent throughout all endpoints.